pyravendb/loaders/query_include_builder.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+from typing import Optional
+
+from pyravendb.loaders.include_builder_base import IncludeBaseBuilder
+from pyravendb.data import document_conventions
+
+
+class QueryIncludeBuilder(IncludeBaseBuilder):
+    def __init__(self, conventions: document_conventions):
+        super(QueryIncludeBuilder, self).__init__(conventions)
+
+    def include_counters(self, *names: str, path: Optional[str] = ""):
```

Comment:
includeCounter seems to be missing, 

please have a look at: https://github.com/ravendb/ravendb-nodejs-client/blob/9e8b405e90eb9ea893e3a1d7ce9558a7a37adc3b/src/Documents/Session/Loaders/QueryIncludeBuilder.ts#L10